src/utils/bitfield.rs

```diff
@@ -19,16 +18,16 @@ pub trait BitGetter: Sized {
     ///
     /// # Arguments
     ///
-    /// * `range` - The range of bits, starting from 0 for the rightmost one.
+    /// * `start` - The range's starting offset.
+    /// * `end` - The range's ending offset.
```

Comment:
Could we precise that end is not included ? In both ConstBitSetter and ConstBitGetter